cmd/scollector/collectors/snmp_ifaces.go

```diff
@@ -64,11 +77,23 @@ func c_snmp_ifaces(community, host string) (opentsdb.MultiDataPoint, error) {
 	if err != nil {
 		return nil, err
 	}
+	t, err := snmp_subtree(host, community, ifType)
```

Comment:
Sure, the tricky thing is that they are basically the "preprocessed"  versions of those, any suggestions?